pyiron_atomistics/atomistics/job/atomistic.py

```diff
@@ -889,3 +896,25 @@ def __dir__(self):
             return hdf5_path.list_nodes()
         else:
             return []
+
+def get_neighbors_traj(struct, positions, cells=None, num_neighbors=20):
+    [n_steps, n_atoms, _] = positions.shape
+    indices, distances = [np.zeros((n_steps, n_atoms, num_neighbors)) for _ in range(2)]
+    vecs = np.zeros((n_steps, n_atoms, num_neighbors, 3))
+    struct = struct.copy()
+    if cells is not None:
```

Comment:
Wouldn’t it be better to do the `if`-statement inside the `for`-loop in order to reduce lines?

Reply:
> 
> 
> Wouldn’t it be better to do the `if`-statement inside the `for`-loop in order to reduce lines?

Yes, but won't it make the code slower since we make the same check at every time step?

Reply:
It’s about whether to perform `if cells is not None` or not, correct? I don’t think that this operation could ever take noticeable amount of time.